src/main/java/gregtech/api/unification/material/Materials.java

```diff
@@ -613,6 +584,41 @@ public static void register() {
+    public static DustMaterial PlatinumGroupSludge = new DustMaterial(381, "platinum_group_sludge", 0x001E00, FINE, 1, of(), DISABLE_DECOMPOSITION );
+    public static DustMaterial HydratedCoal = new DustMaterial(382, "hydrated_coal", 0x464664, ROUGH, 1, of(new MaterialStack(Coal, 8), new MaterialStack(Water, 1)), 0);
+    public static DustMaterial SodiumBicarbonate = new DustMaterial(383, "sodium_bicarbonate", 0x565b96, ROUGH, 1, of(new MaterialStack(Sodium, 1), new MaterialStack(Hydrogen, 1), new MaterialStack(Carbon, 1), new MaterialStack(Oxygen, 3)), 0);
+    public static DustMaterial Collagen = new DustMaterial(384, "collagen", 0x80471C, ROUGH, 1, of(), 0);
+    public static DustMaterial Gelatin = new DustMaterial(385, "gelatin", 0x588BAE, ROUGH, 1, of(), 0);
+    public static DustMaterial Agar = new DustMaterial(386, "agar", 0x4F7942, ROUGH, 1, of(), 0);
+    public static DustMaterial PotassiumDichromate = new DustMaterial(387, "potassium_dichromate", 0xFF084E, DULL, 1, of(new MaterialStack(Potassium, 2), new MaterialStack(Chrome, 2), new MaterialStack(Oxygen, 7)), 0);
+    public static DustMaterial ChromiumTrioxide = new DustMaterial(388, "chromium_trioxide", 0xFFE4E1, DULL, 1, of(new MaterialStack(Chrome, 1), new MaterialStack(Oxygen, 3)), 0);
+    public static DustMaterial AntimonyTrioxide = new DustMaterial(389, "antimony_trioxide", 0xE6E6F0, DULL, 1, of(new MaterialStack(Antimony, 2), new MaterialStack(Oxygen, 3)),0);
+    public static DustMaterial Zincite = new DustMaterial(390, "zincite", 0xFFFFF5, DULL, 1, of(new MaterialStack(Zinc, 1), new MaterialStack(Oxygen, 1)), 0);
+    public static DustMaterial CupricOxide = new DustMaterial(391, "cupric_oxide", 0x0F0F0F, DULL, 1, of(new MaterialStack(Copper, 1), new MaterialStack(Oxygen, 1)), 0);
+    public static DustMaterial CobaltOxide = new DustMaterial(392, "cobalt_oxide", 0x788000, DULL, 1, of(new MaterialStack(Cobalt, 1), new MaterialStack(Oxygen, 1)), 0);
+    public static DustMaterial ArsenicTrioxide = new DustMaterial(393, "arsenic_trioxide", 0xFFFFFF, ROUGH, 1, of(new MaterialStack(Arsenic, 2), new MaterialStack(Oxygen, 3)),0);
+    public static DustMaterial Massicot = new DustMaterial(394, "massicot", 0xFFDD55, DULL, 1, of(new MaterialStack(Lead, 1), new MaterialStack(Oxygen, 1)), 0);
+    public static DustMaterial Ferrosilite = new DustMaterial(395, "ferrosilite", 0x97632A, DULL, 1, of(new MaterialStack(Iron, 1), new MaterialStack(Silicon, 1), new MaterialStack(Oxygen, 3)), 0);
+    public static DustMaterial MetalMixture = new DustMaterial(396, "metal_mixture", 0x502d16, METALLIC, 1, of(), DISABLE_DECOMPOSITION);
+    public static DustMaterial SodiumHydroxide = new DustMaterial(397, "sodium_hydroxide", 0x003380, DULL, 1, of(new MaterialStack(Sodium, 1), new MaterialStack(Oxygen, 1), new MaterialStack(Hydrogen, 1)), 0);
+
+
```

Comment:
Please rearrange these IDs to the proper places.

Reply:
what do you mean?

Reply:
ok, do i shift id's for that?